### PR TITLE
change to declarative sentences. simplify links

### PR DESCRIPTION
**Pull request by smatthewenglish**
The current version is lifted directly from [bitcoin.org](https://bitcoin.org/en/), i.e.

    Bitcoin uses peer-to-peer technology to operate with no central authority or banks; 
    managing transactions and the issuing of bitcoins is carried out collectively by the 
    network.

So as an improvement to readability I propose to change the present perfect progressive such as `managing` and `issuing` to declarative sentences for the form `management` & `issuance`. 

Also I chopped up the last sentence which was kind of long, into two shorter ones with the links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ What is Bitcoin?
...
 anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
+with no central authority: management of transactions and issuance of money are carried
+out collectively by the network. Bitcoin Core is the name of the open source
 software which enables the use of this currency.
 
-For more information, as well as an immediately useable, binary version of
-the Bitcoin Core software, see https://bitcoin.org/en/download, or read the
+An immediately useable binary version of
```
Comment (paveljanik) [inline README.md]: I'm not native language speaker, but the word 'useable' is strange.

**Comment by fanquake** [comment]
ACK adding `the` on line 14 and the link to bitcoin.org on 18, however not convinced the other wording changes are improvements.

**Comment by MarcoFalke** [comment]
@s-matthew-english Agree with @fanquake's feedback.

**Comment by fanquake** [comment]
Closing, as the link in the README is actually already clickable, if someone wants to fix the typo they can do.

**Pull request closed without merging**